installInstructions/buildPETSc.sh

```diff
@@ -0,0 +1,58 @@
+#! /bin/bash
+version= # TODO: PUT THE VERSION OF PETSC YOU WANT TO BUILD HERE (e.g 3.19.5)
```

Comment:
Maybe Marco is referring to adding CLI options for the versions or combinations of opt/debug, real/complex? I'm fine with not adding CLI if the user has to edit the configuration in the file anyway.

Reply:
Yeah exactly, if the file needs to be tweaked anyway then this is fine @A-CGray